app/src/main/java/org/fossasia/openevent/general/about/AboutEventFragment.kt

```diff
@@ -75,26 +73,23 @@ class AboutEventFragment : Fragment(), AppBarLayout.OnOffsetChangedListener {
         rootView.aboutEventContent.text = event.description?.stripHtml()
         val startsAt = EventUtils.getEventDateTime(event.startsAt, event.timezone)
         val endsAt = EventUtils.getEventDateTime(event.endsAt, event.timezone)
+
         rootView.aboutEventDetails.text = EventUtils.getFormattedDateTimeRangeBulleted(startsAt, endsAt)
         rootView.eventName.text = event.name
-    }
-
-    override fun onOffsetChanged(appBarLayout: AppBarLayout, verticalOffset: Int) {
-        val maxScroll = appBarLayout.totalScrollRange
-        val percentage = Math.abs(verticalOffset).toFloat() / maxScroll.toFloat()
+        Picasso.get()
+            .load(event.originalImageUrl)
+            .into(rootView.aboutEventImage)
 
-        if (percentage == 1f && isHideToolbarView) {
-            // Collapsed
-            rootView.detailsHeader.visibility = View.GONE
-            rootView.aboutEventCollapsingLayout.title = title
-            isHideToolbarView = !isHideToolbarView
-        }
-        if (percentage < 1f && !isHideToolbarView) {
-            // Not Collapsed
-            rootView.detailsHeader.visibility = View.VISIBLE
-            rootView.aboutEventCollapsingLayout.title = " "
-            isHideToolbarView = !isHideToolbarView
-        }
+        rootView.appBar.addOnOffsetChangedListener(AppBarLayout.OnOffsetChangedListener { appBarLayout, offset ->
+            val thisActivity = activity
+            if (Math.abs(offset) == appBarLayout.getTotalScrollRange()) {
+                rootView.detailsHeader.visibility = View.GONE
```

Comment:
You can change this to ` rootView.detailsHeader.isVisible = true` .

Reply:
I think it is the same right?

Reply:
No, use .isVisible, as it's in accordance with kotlin code style. What you are using now is in accordance with java code style.